doc/quick_start_guide.rst

```diff
@@ -0,0 +1,129 @@
+.. _quick_start_guide:
+.. include:: ./ext_links.txt
+
+.. |copy| unicode:: U+000A9
+
+.. |trade| unicode:: U+2122
+
+=================
+Quick Start Guide
+=================
+
+Device Drivers
+=================
+
+To start programming data parallel devices beyond CPU, you will need
+an appropriate hardware. The Data Parallel Extension for NumPy* works fine
+on Intel |copy| laptops with integrated graphics. In majority of cases,
+your Windows*-based laptop already has all necessary device drivers installed.
+But if you want the most up-to-date driver, you can always
+`update it to the latest one <https://www.intel.com/content/www/us/en/download-center/home.html>`_.
+Follow device driver installation instructions to complete the step.
+
+
+Python Interpreter
+=================
+
+You will need Python 3.8, 3.9, or 3.10 installed on your system. If you
+do not have one yet the easiest way to do that is to install
+`Intel Distribution for Python*`_. It installs all essential Python numerical
+and machine learning packages optimized for the Intel hardware, including
+Data Parallel Extension for NumPy*.
+If you have Python installation from another vendor, it is fine too. All you
+need is to install Data Parallel Extension for NumPy* manually as shown
+in the next installation section.
+
+
+Installation
+============
+
+Install Package from Anaconda
+---------------------
+
+It is recommended to use conda packages from the ``anaconda.org/intel``
+channel. You will need one of the commands below:
+
+* Conda: ``conda install dpnp``
```

Comment:
Need to use conda-forge channel to install latest version of dpnp package, overwise it will install version dpnp-0.11.1 instead.

Reply:
Done